docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/visionos/index.md

```diff
@@ -0,0 +1,153 @@
+
+let event = DismissWindowEvent(
+    id: "group 1", // window group ID
+    uuid: UUID(), // optional UUID
+    titleKey: "window 1", // optional window title
+    windowStyle: WindowStyle.automatic // optional style of the window group
+)
+tracker.track(event)
+```
+
+*Schema:*
+`iglu:com.apple.swiftui/dismiss_window/jsonschema/1-0-0`.
+
+Determining which events occurred in which window group can be done during modelling using these two event types and timestamps. You could also manually add a window group context entity to tracked events.
+
+In this example, a window group entity is added to an [Ecommerce](docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/ecommerce-tracking/index.md) `ProductViewEvent`.
+
+```swift
+let product = ProductEntity(
+  id: "productId", 
+  category: "category", 
+  currency: "GBP", 
+  price: 100
+)
+let event = ProductViewEvent(product: product)
+let entity = WindowGroupEntity(id: "group_1")
+event.entities.append(entity)
+```
+
+Read more about customized tracking [here](docs/collecting-data/collecting-from-own-applications/mobile-trackers/custom-tracking-using-schemas/index.md).
+
+### Immersive space events
+
+Use the `OpenImmersiveSpaceEvent` and `DismissImmersiveSpaceEvent` to automatically add an immersive space context entity to all events occurring within an immersive space. The entity will identify the immersive space in which the events occurred. This feature is off by default.
+
+```swift
+let tracker = Snowplow.createTracker(
+  namespace: "appTracker", 
+  endpoint: "https://snowplow-collector-url.com"
+) {
+  // configure the automatic immersive space context entity
+  TrackerConfiguration().immersiveSpaceContext(true)
+}
+
+let event = OpenImmersiveSpaceEvent(
+    id: "group 1", // space ID
+    uuid: UUID(), // optional UUID
+    immersionStyle: ImmersionStyle.automatic, // optional
+    upperLimbVisibility: UpperLimbVisibility.hidden // optional
+)
+// all subsequent events, including this one, will have the immersive space entity
+// until DismissImmersiveSpaceEvent is tracked
+tracker.track(event)
+
+// the dismiss event will have the entity
+// but subsequent events will not
+tracker.track(DismissImmersiveSpaceEvent())
+```
+
+*Schemas:*
+`iglu:com.apple.swiftui/open_immersive_space/jsonschema/1-0-0` and 
+`iglu:com.apple.swiftui/dismiss_immersive_space/jsonschema/1-0-0`.
+
+If the immersive space entity autotracking is off (default), the `Open` and `Dismiss` events can still be tracked. However, only the `OpenImmersiveSpaceEvent` will have the entity. The events themselves have no properties.
+
+The VisionOS methods `openImmersiveSpace` and `dismissImmersiveSpace` are asynchronous. We advise that you write your tracking code such that the Snowplow immersive space events await those methods' completion, for accurate tracking.
+
+## VisionOS entities
```

Comment:
This might be better placed under the OOTB data section and linked from here. We also have a `<SchemaProperties />` component that creates a nice representation of the schemas, [see here](https://github.com/snowplow/documentation/blob/main/docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/ootb-data/mobile-lifecycle-events/index.md?plain=1#L52-L55).